src/Tasks/Microsoft.NET.Build.Tasks/ResolveTargetingPackAssets.cs

```diff
@@ -78,14 +80,19 @@ protected override void ExecuteCore()
             }
             else
             {
-                results = Resolve(inputs, Log, BuildEngine4);
+                results = Resolve(inputs, BuildEngine4);
 
                 if (s_allowCacheLookup)
                 {
                     BuildEngine4?.RegisterTaskObject(cacheKey, results, RegisteredTaskObjectLifetime.AppDomain, allowEarlyCollection: true);
                 }
             }
 
+            foreach (var error in results.Errors)
+            {
+                Log.LogError(error);
```

Comment:
Is it worth continuing once errors have been logged?

Reply:
I don't think it hurts anything and it's the way it behaved previously.